tests/roundware/api1/test_commands.py

```diff
@@ -122,9 +124,16 @@ def test_get_asset_info(self):
                     "duraton_in_ms": 5}
         self.assertEquals(expected, get_asset_info(req))
 
+#Jake
     def test_play_asset_in_stream(self):
-        pass
+        req = FakeRequest()
```

Comment:
This unit test changes should be a separate commit and pull request than the fixes for 130.